analytics/views/stats.py

```diff
@@ -54,6 +54,7 @@ def render_stats(
     remote: bool = False,
     analytics_ready: bool = True,
 ) -> HttpResponse:
+    assert isinstance(request.user, UserProfile)
```

Comment:
We've been using `assert request.user.is_authenticated`, which is the standard Django way to write this, for these assertions; mypy with the Django stubs is smart enough to infer this from that statement.